src/Test.UnitTests.BinSkim.Driver/MultithreadedAnalyzeCommandTests.cs

```diff
@@ -129,7 +129,7 @@ public void MultithreadedAnalyzeCommand_ReturnCommonPathRootFromTargetSpecifiers
                 // Testing the same string array in a random order.
                 // This will guarantee that the sorting is working as expected.
                 commonPath = MultithreadedAnalyzeCommand.ReturnCommonPathRootFromTargetSpecifiersIfOneExists(Shuffle(testCase.TargetFileSpecifiers));
```

Comment:
>Shuffle [](http://example.com/codeflow?start=109&length=7)

we are generating a shuffled array. The order might change the expected common path sensitivity (c:/ vs C:/)

This change will guarantee that the order won't matter.